src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -153,8 +151,8 @@ protected function getGridActions()
 
     protected function getBulkActions()
     {
```

Comment:
if you don't need bulk action, you are not forced to implement it (i've already fixed it on my own pr)